backend/templates/home.html

```diff
@@ -1,14 +1,10 @@
 {% extends 'base.html' %}
-
-{% block title %} Home {% endblock %}
-
 {% block content %}
-{% if user.is_authenticated %}
-  Hi {{ user.username }}!
-  <p><a href="{% url 'logout' %}">logout</a></p>
+ 
+    <div class="container">
+      <div class="block">
+        Welocome to Badgeyay!!!
```

Comment:
?

Reply:
Sorry, what do I need to have on the home page? Please tell me so that I can correct it. Do I need to write "hi {{username}}"??

Reply:
What's welocome?

Reply:
It's a dummy welcome message for now as frontend wasn't ready. You tell me the changes, I will do that.

Reply:
https://www.merriam-webster.com/dictionary/welcome

Reply:
okay sorry, I didn't notice the typo. Fixing it right away.

Reply:
Fixed the typo. Please review the PR now @iamareebjamal @rajvaibhavdubey